docs/openapi/components/examples/tile_service_404.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<title>Error</title>
+</head>
+<body>
+<pre>Cannot GET /snthstn</pre>
```

Comment:
Does `snthstn` refer to something specific, or is it just gibberish for the example?